packages/react/src/editor/ElementRenderer.tsx

```diff
@@ -0,0 +1,35 @@
+import { forwardRef, useImperativeHandle, useState } from "react";
+import { createPortal, flushSync } from "react-dom";
+
+/**
+ * A helper component to render a single element to a container so we can subsequently read the DOM / HTML contents
+ *
+ * This is useful so we can render arbitrary React elements (blocks) in the correct context (used by `ReactRenderUtil`)
+ */
+export const ElementRenderer = forwardRef<
+  (node: React.ReactNode, container: HTMLElement) => void
+>((_props, ref) => {
+  const [singleRenderData, setSingleRenderData] = useState<any>();
```

Comment:
Type argument should be `{ node: ReactNode; container: HTMLElement } | undefined` I think